common/system/filtering/altitude_kalman.h

```diff
@@ -163,6 +164,12 @@ class AltitudeKalman {
 
   void setDt(const double dt);
 
+  void calculateDt();
+
+  double calculateGain(int g);
```

Comment:
What is `g`? Probably good to add a docstring here (and to all the functions) to specify what they do and what the parameters are. See https://github.com/AerospaceNU/stm32-avionics/blob/0e5da221fc4a483ade83f1d7953fb68a68a86224/common/system/filtering/filters.h for a good example of docstring comments.